src/feature.rs

```diff
@@ -31,12 +31,13 @@ pub trait Ext: Sized {
     ///   Scenario Outline: eating
     ///     Given there are <start> cucumbers
     ///     When I eat <eat> cucumbers
+    ///      | <eat> |
```

Comment:
Didn't you forget a column name here? This example is quite unobvious. Can we make it more meaningful?